content/en/user-guide/aws/docdb/index.md

```diff
@@ -231,16 +232,125 @@ the [MongoDB documentation](https://www.mongodb.com/docs/).
 ### Connect to DocumentDB using Node.js Lambda
 
 {{< alert title="Important" color="success">}}
-You need to set `DOCDB_PROXY_CONTAINER=1` when starting LocalStack to be able to use the `LOCALSTACK_HOSTNAME` in lambda to connect to the instance.
+You need to set `DOCDB_PROXY_CONTAINER=1` when starting LocalStack to be able to use the `localhost.localstack.cloud` in lambda to connect to the instance.
 {{< /alert>}}
 
-The code snippet is a Node.js lambda.
-In this sample we assume that we have a secret where the DocumentDB credentials are stored.
-* The secret arn is passed as env `SECRET_NAME`.
-* The Lambda reads the connection details and connects to the running DocumentDB, which runs in proxied docker container. 
-* As we run inside lambda, we change the host to the `LOCALSTACK_HOSTNAME` to connect to the container.
-* A helper function makes sure the username and password are encoded correctly for the connection URI.
-* We run some sample commands to verify the connection works.
+In this sample we will use Node.js lambda function to connect to a DocumentDB. 
+For the mongo-db connection we will use the `mongodb` lib.
+Please note, that this sample is only for demo purpose, e.g., we will set the credentials as environment variables to the lambda function.
+
+In a best-practise sample you would use a secret instead.
+We included a snippet at the very end.
+
+#### Create the DocDB Cluster with a username and password
```

Comment:
I think it is great to align the DB creation with the sample code snippet and make it more self-contained.

One concern: The current page structure gives the impression that after following the first parts of getting started, the `Nodejs` section shows how to connect to the same cluster (using the employees, customers theme).

Reply:
one way or another we want to show both options: once without user/pw, and one with. 
IMO this way it's less confusing 🤷‍♀️